docs/sources/clients/promtail/configuration.md

```diff
@@ -1756,6 +1759,26 @@ scrape_configs:
         target_label: 'container'
 ```
 
+## limit_config
```

Comment:
This looks good @KMiller-Grafana, and we should align this with the flags registered here:
https://github.com/grafana/loki/blob/main/clients/pkg/promtail/limit/config.go#L14-L19